runtime/gc_modron_standard/RootScannerReadBarrierVerifier.cpp

```diff
@@ -94,6 +94,15 @@ MM_RootScannerReadBarrierVerifier::scanClassStaticSlots(MM_EnvironmentBase *env)
 		while (NULL != (clazz = classHeapIterator.nextClass())) {
 
 			volatile omrobjectptr_t *slotPtr = NULL;
+			/*
```

Comment:
this shouldnt be here. Default values should not be scanned as part of class statics

Reply:
Should I remove it completely or move it to somewhere else other than here?

Reply:
remove it completely